icons/cup-to-go.svg

```diff
@@ -0,0 +1,16 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M3 7h18" />
+  <path d="m19.2 11 .8-4-.8-4c-.1-.5-.6-1-1.2-1H6c-.6 0-1.1.4-1.2 1L4 7l.8 4" />
+  <path d="M19 18H5l-1-7h16Z" />
+  <path d="m6.2 18 .6 3c.1.5.6 1 1.2 1h8c.6 0 1.1-.4 1.2-1l.6-3" />
```

Comment:
I think this feels a bit clunky, I'd slim it down a bit:
![image](https://github.com/lucide-icons/lucide/assets/17746067/5482b1f0-9bd0-4459-a8c3-11500228eeb2)

```suggestion
  <path d="M4 6h16" />
  <path d="M17.63 17H6.38" />
  <path d="M5.63 11H18.38" />
  <path d="M8.24 2h7.52a2 2 0 0 1 1.8 1.1L19 6l-1.78 14.25A2 2 0 0 1 15.23 22H8.77a2 2 0 0 1-1.99-1.75L5 6l1.45-2.9A2 2 0 0 1 8.24 2Z" />
```